WordPress/Classes/Services/PostService.h

```diff
@@ -48,7 +52,7 @@ typedef void(^PostServiceSyncFailure)(NSError *error);
  @param success A success block
  @param failure A failure block
  */
-- (void)syncPostsOfType:(NSString *)postType
+- (void)syncPostsOfType:(PostServiceType)postType
```

Comment:
`PostServiceType` sounds like a bad naming choice here, considering the caller is referring to the post type here, not the service type.

I understand that this method internally picks the service base on the post type, but the name should still be `PostType`, as the caller is only providing post information and is unaware of services being used internally by this class.